engine/execution/state/state_test.go

```diff
@@ -178,7 +178,7 @@ func TestExecutionStateWithTrieStorage(t *testing.T) {
 
 		// update value and get resulting state commitment
 		view2 := es.NewView(sc2)
-		err = view2.Delete(registerID1, "")
+		err = view2.Set(registerID1, "", []byte("orange"))
```

Comment:
How come this test wasn't changed to test `view2.Set(registerID1, "", nil)`?

Reply:
I wasn't sure if cadence is allowed to set the value to nil, so I picked an arbitrary non-nil value instead.  I'll change this to nil.